test/runtests.jl

```diff
@@ -613,14 +613,19 @@ end
 
 @testset "EquiRect" begin
 
-    θspan = deg2rad.((140,110))
-    φspan = deg2rad.((-10,90))
+    θspan = (π/2 .- deg2rad.((-40,-70)))
+    φspan = deg2rad.((0,120))
     Cℓ = camb()
 
-    @testset "Nside=$Nside" for Nside in Nsides
+    @testset "Nside = $Nside" for Nside in Nsides_big
 
         local f0, f2, Cf0, Cf2
 
+        @test begin
+            proj = ProjEquiRect(Ny=128, Nx=128, θspan=(π/2 .- deg2rad.((-40,-70))), φspan=deg2rad.((-50,50)))
+            proj.Ny == proj.Nx == 128
+        end
```

Comment:
This is the 3G θspan and φspan, and we should be able to create a map with these particular values (the Nside is made up, that doesn't matter). I dont mind if we have to flip the order of θspan, but φspan should be able to straddle zero. Also, we need to make it so any errors about φspan not being a fraction of 2π should only come if we actually try and create a BlockDiagEquiRect with this projection. For reasons I dont understand, this currently truncates the θspan to 19 pixels. Note the code I put in the constructor above (which causes this test failure) is copied from your scratch equirect1.jl. I could use your help sorting this one out.

Reply:
>  For reasons I dont understand, this currently truncates the θspan to 19 pixels.

This is fixed now. The issue was that, for consistency across options `{:equiθ, :equicosθ, :healpix}`, I had decided that `N` would control the number of grid points within the full range `(0,pi)` which would get subsetted to the requested `θspan`. In hindsight this was clumsy. Now `N` does what you would expect, except in the `:healpix` case where `N` is equivalent to `Nside` and it will not necessarily match the final number of grid points within  `θspan`. I've added a warning to the user when `:healpix` is used. Let me know if you think we should have different behavior.

Reply:
That looks good, thanks! I think that behavior is reasonable too with healpix. Looks like a few more tests pass now :tada: